src/main/java/net/sf/jabref/logic/layout/format/EntryTypeFormatter.java

```diff
@@ -0,0 +1,22 @@
+package net.sf.jabref.logic.layout.format;
+
+import net.sf.jabref.logic.TypedBibEntry;
+import net.sf.jabref.logic.layout.LayoutFormatter;
+import net.sf.jabref.model.database.BibDatabaseMode;
+import net.sf.jabref.model.entry.BibEntry;
+
+
+public class EntryTypeFormatter implements LayoutFormatter {
+
+    @Override
+    public String format(String fieldText) {
```

Comment:
Could you state that here the entryType as string is expected? Maybe rename the parameter form `fieldText` to `entryType`?